naga/src/front/wgsl/parse/directive/language_extension.rs

```diff
@@ -60,9 +60,25 @@ impl LanguageExtension {
     }
 }
 
-/// A variant of [`LanguageExtension::Implemented`].
+// A variant of [`LanguageExtension::Implemented`].
+/// Enum of all WGSL extensions that are implemented
+/// WGSL spec.: <https://www.w3.org/TR/WGSL/#language-extensions-sec>
 #[derive(Clone, Copy, Debug, Eq, Hash, PartialEq)]
-pub(crate) enum ImplementedLanguageExtension {}
+pub enum ImplementedLanguageExtension {}
```

Comment:
One final thought - any reason not to make this a bitflags too?

Reply:
> One final thought - any reason not to make this a bitflags too?

Because of the way it's curretnly used in naga and this enum is complementary to `UnimplementedLanguageExtension`.